packages/ember-glimmer/lib/ember-metal-views/index.js

```diff
@@ -13,6 +15,37 @@ class DynamicScope {
   }
 }
 
+const RENDERED_VIEWS = [];
```

Comment:
should name this clearly, like RENDERED_ROOTS; want to ensure people realize this are roots and not any view.